docs/source/extensions/overview_extensions.rst

```diff
@@ -21,11 +21,15 @@ The following extensions are officially supported and were developed as part
 of the core project:
 
 - :doc:`extension_covid19`: Make literature on COVID-19 available in ASReview.
-- :doc:`extension_visualization`: Plotting functionality for state files produced by ASReview.
-- :doc:`extension_wordcloud`: Create a visual impression of the contents of datasets via a wordcloud.
-- ``asreview-statistics``: Tool to give some basic properties of a dataset, such as number of
-  papers, number of inclusions. `GitHub <https://github.com/asreview/asreview-statistics>`__
-- ``asreview-hyperopt``: Optimize the hyperparameters of the models in ASReview. `GitHub <https://github.com/asreview/asreview-hyperopt>`__
+- :doc:`extension_visualization`: Plotting functionality for state files 
+  produced by ASReview.
+- :doc:`extension_wordcloud`: Create a visual impression of the contents of 
+  datasets via a wordcloud.
+- `ASReview-statistics <https://github.com/asreview/asreview-statistics>`__: 
+  Tool to give some basic properties of a dataset, such as number of papers, 
+  number of inclusions. 
+.. - `ASReview-vocab-extractor <https://github.com/asreview/asreview-extension-vocab-extractor>`__: This extension adds two feature extractors that extract vocabulary and vector matrices during simulation phases.
+.. - ``asreview-hyperopt``: Optimize the hyperparameters of the models in ASReview. `GitHub <https://github.com/asreview/asreview-hyperopt>`__
```

Comment:
Commented? Shouldn't this be included?

Reply:
I figured its outdated and "❗Highly experimental❗". Ill include it